src/components/LibraryDetailPage.tsx

```diff
@@ -79,7 +80,7 @@ export class LibraryDetailPage extends React.Component<LibraryDetailPageProps, L
     let libraryStage = this.props.fullLibrary.stages.library_stage;
     let registryStage = this.props.fullLibrary.stages.registry_stage;
     this.props.updateColor([libraryStage, registryStage]);
-    this.setState({ libraryStage, registryStage });
+    this.setState(Object.assign(this.state, { libraryStage, registryStage }));
```

Comment:
Why this update?

Reply:
Because of something that I subsequently got rid of.  I'll revert this!

Reply:
Alright. It also doesn't do anything since `this.state` only has two values and you are overriding those values anyway.

Reply:
I know--the thing I got rid of involved having a third value :)